aeon_crud.proto

```diff
@@ -43,6 +43,24 @@ service CRUDService {
 // Transactionally executes a set of read and write operations.
 
 message ExecuteRequest {
+  // Message queue push section in a transaction request.
+  message Push {
+    // Definition of messages pushed to a shard-local queue.
+    message Message {
+      // Topic of the message.
+      string topic = 1;
+      // Data of the message.
+      Value data = 2;
+      // Time to live of the message.
+      double ttl = 3;
+    }
+    // Push function.
```

Comment:
What arguments does this function take? In what language is it written? What is it supposed to return? Please write comments so that a user who isn't familiar with the implementation can figure out how to use this API.